lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -69,7 +69,7 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 		if (options && options.baseUri) {
 			return `${RuntimeGlobals.baseURI} = ${JSON.stringify(options.baseUri)};`;
 		}
-		return `${RuntimeGlobals.baseURI} = (document && document.baseURI) || self.location.href;`;
+		return `${RuntimeGlobals.baseURI} = (typeof document === 'object' && document.baseURI) || self.location.href;`;
```

Comment:
Let's use `typeof document !== "undefined"`, we prefer to check on undefined/not undefined instead object